### DISCUSSION TITLE
Limit user requests

### DISCUSSION THREAD

**Discussion by Foebik** [sct/overseerr#1248] [Feature Requests]
Sorry if this has been asked and answered before, but I didnt see it. Is it possible to limit the number of TV shows or Movies a user can request? Or is this a feature that is looking to go in further down the road?

Example:
2 TV series per week (or whatever time frame)
5 movies per week  (or whatever time frame)

Thanks in advance

**Comment by sct** [comment]
This is already in progress: https://github.com/sct/overseerr/pull/945

**Reply by Foebik** [reply]
TY @sct
